src/bors/handlers/refresh.rs

```diff
@@ -325,15 +325,11 @@ timeout = 3600
                 })
                 .await?;
             tester.refresh_prs().await;
-            assert_eq!(
-                tester
-                    .db()
-                    .get_pull_request(&default_repo_name(), pr.number)
-                    .await?
-                    .unwrap()
-                    .pr_status,
-                PullRequestStatus::Open
-            );
+            tester
```

Comment:
Hmm, I'm not actually sure if these are 100% equivalent. We want to make sure here that the PR has the specific state already after the refresh is done, not sometime in the future. Of course, the `await` might cause some minor race conditions on its own, but I still think that we shouldn't need `wait_for_pr` in these situations.

I agree that the code is a bit verbose though, so I'm fine with adding a function to `tester`, something like `get_pr_from_db(pr_number)` that loads a PR with a given number from the default repo and unwraps the result.